Tryouts/Visuals/Windows/WindowsVisualUtils/EmbeddedWebView.cs

```diff
@@ -0,0 +1,166 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Platform;
+using Microsoft.Web.WebView2.Core;
+using Microsoft.Web.WebView2.WinForms;
+
+namespace MorganStanley.ComposeUI.Tryouts.Visuals.Avalonia.WindowsVisualUtils
```

Comment:
Inconsistent namespace: in the namespace you have Tryouts.Visuals.Avalonia. but the file is in Tryouts\Visuals\Windows\..

Reply:
fixed